mcsema/BC/Instruction.cpp

```diff
@@ -265,18 +265,16 @@ llvm::Value *InstructionLifter::LiftAddressOperand(
     if ( map_it != ctx.cfg_inst->stack_var->refs.end()) {
       auto var_offset = llvm::ConstantInt::get(word_type, static_cast<uint64_t>(map_it->second), true);
       base = ir.CreateAdd(base, var_offset);
-      LOG(INFO) << "Lifting stack variable access at : " << std::hex << map_it->first
+      LOG(INFO)
+      	  << "Lifting stack variable access at : " << std::hex << map_it->first
```

Comment:
I think you put a tab in here. Please adjust your text editor / IDE to always use spaces instead of tabs.